docs/remixd.md

```diff
@@ -34,9 +34,11 @@ folder.
 There is an option to run remixd in read-only mode, use `--read-only` flag.
 
 ### Warning!
-`remixd` provides `full read and write access` to the given folder for `any
+- `remixd` provides `full read and write access` to the given folder for `any
 application` that can access the `TCP port 65520` on your local host.
 
+- To minimize the risk, Remixd can only bridge between your filesystem and Remix IDE URLS.
```

Comment:
We recommend using remixd with from one of the supported origins listed [here](https://gist.githubusercontent.com/EthereumRemix/091ccc57986452bbb33f57abfb13d173/raw/3367e019335746b73288e3710af2922d4c8ef5a3/origins.json).